internal/core/graphics.rs

```diff
@@ -46,9 +46,9 @@ pub(crate) mod bitmapfont;
 pub use self::bitmapfont::*;
 
 #[cfg(feature = "std")]
-mod fps_counter;
+mod rendering_metrics_collector;
 #[cfg(feature = "std")]
-pub use fps_counter::*;
+pub use rendering_metrics_collector::*;
```

Comment:
I think this is a bit unfortunate to have such a specific detail "leaking" in the whole API. But i think we should not at least leak the whole module in the `graphics` namespace.  I guess it's fine to keep `rendering_metrics_collector` as a pub module.